README.md

```diff
@@ -22,11 +22,9 @@ Create a campus guide application to help students locate their classrooms.
 
 ## Installation Instructions
 1. Download and install [Android Studio](https://developer.android.com/studio)
-2. [Get an API key for Google Maps](https://developers.google.com/maps/documentation/android-sdk/get-api-key)
-3. Clone the repository onto your computer
-4. Create a copy of the file `app\src\debug\res\values\google_maps_api.example.xml` in the same folder.
-5. Rename that file `google_maps_api.xml`
-6. The file contains a single `<string>` tag. Change the `name` property from `google_maps_key_ex` to `google_maps_key`. Change the contents of the tag from `google_maps_key` to the value of your API key (it starts with `AIza`)
-7. Perform steps 4 through 6 with the file `app\src\release\res\values\google_maps_api.example.xml`. This step is only necessary to build the app in Release configuration.
-8. Open the project in Android Studio
-9. Run the project using an emulator or using a connected android device.
+1. [Get an API key for Google Maps](https://developers.google.com/maps/documentation/android-sdk/get-api-key)
+1. Clone the repository onto your computer
+1. Create a file `app\src\main\res\values\google_maps_api.xml`.
```

Comment:
As far as I know:
To run the app in debug mode, you need the file in `app\src\debug`
To run the app in production mode, you need the file in `app\src\release`
Not sure why it would need to be in main...

Reply:
Those directories don't exist in the repository's directory structure...

Reply:
They don't exist in the repo because they're gitignored
They do however exist locally (since they contain the API key)

Reply:
No, i don't have them locally. My api key is in the app\src\main\... dir